libmamba/tests/src/catch-utils/msvc_catch_byte.cpp

```diff
@@ -0,0 +1,22 @@
+#ifdef _WIN32
+
+// Catch compiled on `conda-forge` for MSVC doesn't support outputting `std::byte`.
+// So we have to define StringMaker for it ourselves.
+// The declaration is present though, so this only causes link errors.
```

Comment:
Has this problem been reported upstream? If not can we report it?

Reply:
I think I found relevant issue upstream: https://github.com/conda-forge/vc-feedstock/issues/45